volatility3/framework/symbols/windows/extensions/__init__.py

```diff
@@ -351,17 +351,38 @@ def dereference(self) -> interfaces.objects.ObjectInterface:
 class DEVICE_OBJECT(objects.StructType, pool.ExecutiveObject):
     """A class for kernel device objects."""
 
-    def get_device_name(self) -> str:
-        header = self.get_object_header()
-        return header.NameInfo.Name.String  # type: ignore
-
+    def get_device_name(self) -> Union[str, interfaces.renderers.BaseAbsentValue]:
+        """Get device's name from the object header."""
+        try:
+            header = self.get_object_header()
+            return header.NameInfo.Name.String  # type: ignore
+        except(ValueError):
+            return renderers.UnparsableValue()
+
+    def get_attached_devices(self) -> interfaces.objects.ObjectInterface:
```

Comment:
This is an addition to the core extensions, as such we should be bumping the `VERSION_PATCH` number, and the `_required_framework_version` should be incremented to require that version at a minimum.  All other plugins should be fine since the change is only additive.

Reply:
This also actually returns a `Generator[ObjectInterface, None, None]` I think, rather than just a single `ObjectInterface`?

Reply:
Yes, I will check this and change it. According to the policy, it will be updated to `2.0.3`. I will change the minimum version of DeviceTree to the same.

Reply:
Yep, perfect, thanks.  5:)